person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age:, parent_permission: true, name: 'Unknow')
```

Comment:
- [ ] I was having errors while creating a person object.

  - I think we should have `age`, instead of `age:` (notice the colon)
  - I think we should have `name = 'Unknow'`, instead of `name: 'Unknow'`
  - I think the `name` parameter should come before the `parent_permission`. Seems to throw `unexpected end` error

  In conclusion, we should have something like this:
  ```
  def initialize(age, name = 'Unknow', parent_permission: true)
    ...
  ```

  ![person-err](https://user-images.githubusercontent.com/36542986/179575738-887ec681-ed3a-49e2-ac70-f4a600e1cd17.jpg)

Reply:
Hi, @TedLivist to create an object with a class that uses that notation you have to create an object like this: 
pers = Person.new(age: 18, parent_permission: true, name:"Teddy"). Same for student and teacher objects.

Reply:
Hi @Santiago220991 alright then. I tried your notation and it works. Sorry for the inconvenience. Kindly request another review.